pkg/apis/cyclone/v1alpha1/workflow_run.go

```diff
@@ -39,8 +39,38 @@ type WorkflowRunSpec struct {
 	Stages []ParameterConfig `json:"stages"`
 	// Execution context which specifies namespace and PVC used
 	ExecutionContext *ExecutionContext `json:"executionContext"`
+	// PresetVolumes will mount to pod of all stages. There are some kinds of preset volumes supported to mount to
+	// a workload pod: HostPath, PV. With HostPath, you can mount your host timezone file to all stages to ensure
+	// they are in the same timezone with the host, and you cna mount certificate file to stages for authentication;
+	// With PV, you can share things amount stages.
+	// Notes:
+	// - Only useful with Pod type stages, Delegation type stages will ignore;
+	// - HostPath type volume will only have Read access to the host.
+	PresetVolumes []PresetVolume `json:"volumes,omitempty"`
 }
 
+// PresetVolume defines a preset volume
+type PresetVolume struct {
+	// Type of the volume
+	Type PresetVolumeType `json:"type"`
+	// VolumePath is the path of volume. with HostPath type volume, it is the path of the host,
+	// with PV type volume, it is the path of the pv.
+	VolumePath string `json:"volumePath"`
+	// Path in the pod where the VolumePath will mount to
+	Path string `json:"path"`
+}
+
+// PresetVolumeType is one type of stages preset mount volumes, there are some kinds of preset volume
+// supported to mount to a workload pod: HostPath, PV.
+type PresetVolumeType string
```

Comment:
PresetVolumeType is type of the volume. Types 'HostPath', 'PV' are supported now.